### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by syl-ilumb**
Corrected installation options from https://www.sylabs.io/singularity/download/ to https://www.sylabs.io/singularity/ to account for recent updates to the Sylabs.io website.

**Description of the Pull Request (PR):**

Corrected installation options from https://www.sylabs.io/singularity/download/ to https://www.sylabs.io/singularity/ to account for recent updates to the Sylabs.io website.

**This fixes or addresses the following GitHub issues:**

- Fixes #

Incorrect link

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](../CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](../CHANGELOG.md) if necessary according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](../CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

**Review by GodloveD — CHANGES_REQUESTED** [review]
I think the link should point here:
https://www.sylabs.io/guides/3.0/user-guide/installation.html

It's probably also useful for members of the open source community to be aware of the PRO offering, so maybe we should explicitly add that as another link as well.

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Check out [who is using Singularity](https://www.sylabs.io/singularity/whos-usin
 
 ## Getting Started with Singularity
 
-To install Singularity from source, see the [installation instructions](INSTALL.md). For other installation options, see [our website](https://www.sylabs.io/singularity/download/).
+To install Singularity from source, see the [installation instructions](INSTALL.md). For other installation options, see [our website](https://www.sylabs.io/singularity/).
```
Comment (ArangoGutierrez) [inline README.md]:
```suggestion
To install Singularity from source, see the [installation instructions](INSTALL.md). For other installation options, see [our website](https://www.sylabs.io/guides/3.0/user-guide/installation.html).
```

**Review by ArangoGutierrez — APPROVED** [review]

**Comment by ArangoGutierrez** [comment]
PING -> @GodloveD

**Review by GodloveD — APPROVED** [review]

**Comment by GodloveD** [comment]
Thanks @syl-ilumb and @ArangoGutierrez

**Pull request merged**